### PR TITLE
proposed security.md from #202

### PR DESCRIPTION
**Pull request by greggles**
It's a start, at least.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Security Policy
+
+File any security concerns with Docker in this Github project as an Issue.
+
+File any security concerns with Drupal following the [Drupal security issue process](https://www.drupal.org/docs/develop/issues/issue-procedures-and-etiquette/reporting-a-security-issue).
+
```
Comment (tianon) [inline SECURITY.md:6]:
```suggestion
If you believe you have found a security issue with Drupal itself, please follow [the Drupal security issue process](https://www.drupal.org/docs/develop/issues/issue-procedures-and-etiquette/reporting-a-security-issue).

If your issue is specific to the Dockerization / container image, [see the instructions in the umbrella "Official Images" `SECURITY.md` file](https://github.com/docker-library/official-images/tree/HEAD/SECURITY.md) for how to reach the @docker-library organization maintainers privately.
```

**Review by tianon — COMMENTED** [review]
Thanks for kicking this off!

We've merged https://github.com/docker-library/official-images/pull/11454 to have a central place for security guidance for all the @docker-library repositories, so I've got a suggested wording change to point to that instead. :+1:

**Comment by greggles** [comment]
Ah yes, your suggestion is a dramatic improvement to what I had proposed. Thanks!

**Comment by tianon** [comment]
Nice, thank you again!

**Pull request merged**